### ISSUE TITLE
The server is unable to execute your request.

### ISSUE THREAD

**Issue by Grorom** [nextcloud/registration#285]
Hi,

Thanks for this great app !!!

### Steps to reproduce
1. update Registration to 1.10
2. register with apps/registration/
3. enter an email
4. enter the code you received 
5. enter your name and password
6. see the error (in french, sorry) : 
```
Le serveur est incapable d'exécuter votre requête.

Si cela se reproduit, veuillez envoyer les détails techniques ci-dessous à l'administrateur du serveur.

Le fichier journal du serveur peut fournir plus de renseignements.
Renseignements techniques

    Adresse distante : x.x.x.x
    ID de la demande : ZlD6Wa7uN3l0JuslbBlI
```

### Expected behaviour
To register the user in the db (as it was possible with the registration 1.00)

### Actual behaviour
The error message paste above and no registration in the database.

### Server configuration
**Operating system**:
Ubuntu 20.04.2 LTS
**Web server:**
apache2
**Database:**
mysql 10.3.25
**PHP version:**
7.4.3
**Nextcloud version:** 
21.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:**
fresh install of Nextcloud 21 a month ago

**Where did you install Nextcloud from:**
https://nextcloud.com/install/#instructions-server

**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>Enabled:
  - accessibility: 1.7.0
  - calendar: 2.2.0
  - cloud_federation_api: 1.4.0
  - comments: 1.11.0
  - contactsinteraction: 1.2.0
  - dashboard: 7.1.0
  - dav: 1.17.1
  - documentserver_community: 0.1.9
  - federatedfilesharing: 1.11.0
  - federation: 1.11.0
  - files: 1.16.0
  - files_pdfviewer: 2.1.0
  - files_rightclick: 1.0.0
  - files_sharing: 1.13.1
  - files_trashbin: 1.11.0
  - files_versions: 1.14.0
  - files_videoplayer: 1.10.0
  - firstrunwizard: 2.10.0
  - logreader: 2.6.0
  - lookup_server_connector: 1.9.0
  - nextcloud_announcements: 1.10.0
  - notifications: 2.9.0
  - notify_push: 0.1.6
  - oauth2: 1.9.0
  - password_policy: 1.11.0
  - privacy: 1.5.0
  - provisioning_api: 1.11.0
  - recommendations: 1.0.0
  - registration: 1.1.0
  - richdocuments: 4.0.4
  - richdocumentscode: 6.4.705
  - serverinfo: 1.11.0
  - settings: 1.3.0
  - sharebymail: 1.11.0
  - spreed: 11.1.1
  - support: 1.4.0
  - survey_client: 1.9.0
  - systemtags: 1.11.0
  - text: 3.2.0
  - theming: 1.12.0
  - twofactor_backupcodes: 1.10.0
  - updatenotification: 1.11.0
  - user_status: 1.1.1
  - viewer: 1.5.0
  - weather_status: 1.1.0
  - workflowengine: 2.3.0
Disabled:
  - activity
  - admin_audit
  - bruteforcesettings
  - contacts
  - deck
  - encryption
  - files_external
  - mail
  - notes
  - onlyoffice
  - photos
  - user_ldap

</details>

**The content of config/config.php:**
<details>
<summary>Config report</summary>

  'instanceid' => '',
  'passwordsalt' => '',
  'secret' => '',
  'trusted_domains' => 
  array (
    0 => 'xxxxxxx.fr',
    1 => 'xxxxxxx.org',
    2 => 'xxxxxxx.com',
  ),
  'datadirectory' => '/media/hardDrive/ncdata-sampl/',
  'dbtype' => 'mysql',
  'version' => '21.0.0.18',
  'overwrite.cli.url' => '',
  'dbname' => '',
  'dbhost' => 'localhost',
  'dbport' => '',
  'dbtableprefix' => 'oc_',
  'mysql.utf8mb4' => true,
  'dbuser' => '',
  'dbpassword' => ',
  'installed' => true,
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'default_phone_region' => 'FR',
  'app_install_overwrite' => 
  array (
    0 => 'richdocumentscode',
    1 => 'onlyoffice',
  ),
  'mail_from_address' => 'contact',
  'mail_smtpmode' => 'smtp',
  'mail_sendmailmode' => 'smtp',
  'mail_domain' => 'xxxxxxx',
  'mail_smtpauthtype' => 'LOGIN',
  'mail_smtphost' => 'smtp-zose.yulpa.io',
  'mail_smtpport' => '465',
  'mail_smtpauth' => 1,
  'mail_smtpname' => 'xxxxx@xxxxx',
  'mail_smtppassword' => 'xxxxx',
  'mail_smtpsecure' => 'ssl',
  'skeletondirectory' => '',
  'maintenance' => false,
  'htaccess.RewriteBase' => '/',
  'trusted_proxies' => 
  array (
    0 => '127.0.0.1',
    1 => '::1',
  ),
  'theme' => '',
  'loglevel' => 2,
  'log_type' => 'file',
  'logfile' => '/var/www/nextcloud-sampl/nextcloud.log',
  'logfilemode' => 0640,
  'logdateformat' => 'F d, Y H:i:s', 
);

</details>

**Are you using external storage, if yes which one:** local

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:**
Firefox 87, Chromium 89.0.4389.114
**Operating system:**
Kubuntu 20.10, Windows 10

### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
{"reqId":"X0Nn0OiXLcg6rEs0UF3p","level":3,"time":"April 04, 2021 14:06:47","remoteAddr":"192.168.1.1","user":"--","app":"index","method":"POST","url":"/apps/registration/register/E65NkoA4kQr93qffTe9R6MAHdf2ofz6A/gBDnw7XpSN","message":{"Exception":"Exception","Message":"Argument 3 passed to OCA\\Registration\\Controller\\RegisterController::submitUserForm() m>
{"reqId":"3VPr1giqlR6actOq9Dt4","level":3,"time":"April 04, 2021 14:14:29","remoteAddr":"192.168.1.1","user":"--","app":"index","method":"POST","url":"/apps/registration/register/E65NkoA4kQr93qffTe9R6MAHdf2ofz6A/gBDnw7XpSN","message":{"Exception":"Exception","Message":"Argument 3 passed to OCA\\Registration\\Controller\\RegisterController::submitUserForm() m>
{"reqId":"ZlD6Wa7uN3l0JuslbBlI","level":3,"time":"April 04, 2021 14:16:34","remoteAddr":"192.168.1.1","user":"--","app":"index","method":"POST","url":"/apps/registration/register/E65NkoA4kQr93qffTe9R6MAHdf2ofz6A/gBDnw7XpSN","message":{"Exception":"Exception","Message":"Argument 3 passed to OCA\\Registration\\Controller\\RegisterController::submitUserForm() m>
```
</details>

**Comment by nickvergessen** [comment]
See #283 
I just released 1.1.1 to fix it

**Comment by Grorom** [comment]
wow ! Thanks for the quickness and the work ! It works perfectly !
